ansible/roles/vm_set/library/kvm_port.py

```diff
@@ -29,17 +29,17 @@ def main():
 
     try:
         output = subprocess.check_output(
-                "virsh domiflist %s" % vmname, 
-                env={"LIBVIRT_DEFAULT_URI": "qemu:///system"}, 
-                shell=True)
+                "virsh domiflist %s" % vmname,
+                env={"LIBVIRT_DEFAULT_URI": "qemu:///system"},
+                shell=True).decode('utf-8')
     except subprocess.CalledProcessError:
         module.fail_json(msg="failed to iflist dom %s" % vmname)
 
     mgmt_port = None
     fp_ports = []
 
     for l in output.split('\n'):
-        fds = re.split('\s+', l)
+        fds = re.split('\s+', l.lstrip())
```

Comment:
due to virsh output format change, there will be leading whitespaces, removing them.